doc/admin/install/google_cloud.md

```diff
@@ -24,23 +24,38 @@ If you're just starting out, we recommend [installing code-server locally](../..
 ```
 gcloud compute ssh --zone [region] [instance name]
 ```
-2. Download the binary using the link we copied to clipboard
+
+2. Find and download the latest Linux release from this URL:
+```
+https://github.com/codercom/code-server/releases/latest
+```
+
+3. Extract the downloaded tar.gz file with this command, for example:
 ```
-wget https://github.com/codercom/code-server/releases/download/0.1.4/code-server-linux
+tar -xvzf code-server-1.31.1-100-linux-x64.tar.gz
 ```
-3. Make the binary executable if you run into any errors regarding permission:
-  ```
-  chmod +x code-server-linux
-  ```
+
+4. Make the binary executable if you run into any errors regarding permission:
+```
+chmod +x code-server-linux
```

Comment:
This has changed a bit; the packaged binary name will always be `code-server` now.